src/inverseRobot.js

```diff
@@ -7,7 +7,16 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const revert = {};
+
+  for (const key in robot) {
+    if (robot[key] in revert) {
+      return null;
+    }
+    revert[robot[key]] = key;
```

Comment:
Would be good to add empty line here for better readability. In this case it's ok without it, but it’s a good habit to have when code becomes bigger
```suggestion
    if (robot[key] in revert) {
      return null;
    }
    
    revert[robot[key]] = key;
```